src/nativeAssetManager.js

```diff
@@ -227,12 +271,43 @@ export function newNativeAssetManager(win) {
     }
 
     if (data.message === 'assetResponse') {
-      const body = win.document.body.innerHTML;
-      const newHtml = replace(body, data);
+      const body = win.document.body.innerHTML,flag = (typeof win.pbNativeData !== 'undefined');
 
-      win.document.body.innerHTML = newHtml;
-      callback && callback();
-      win.removeEventListener('message', replaceAssets);
+      if((data.hasOwnProperty('rendererUrl') && data.rendererUrl) || (flag && win.pbNativeData.hasOwnProperty('rendererUrl'))) {
+        if(win.renderAd){
```

Comment:
some lint issues (we do not have automatic linter on puc code yet)

Maybe run the entire code on your local with your eslint plugin and do the fixes?

Reply:
Updated